src/components/Instructions.component.js

```diff
@@ -1,18 +1,36 @@
 import React, { Component } from 'react'
-// TODO: uncomment this for Part 1
-// import PropTypes from 'prop-types'
+import PropTypes from 'prop-types'
 import './../styles/instructions.css'
 
 class Instructions extends Component {
+  static propTypes = {
+    shouldDisplayImage: PropTypes.bool,
+    items: PropTypes.arrayOf(PropTypes.string),
+  }
   render() {
     return (
       <div className="instructions">
         Follow the instructions on the README to get started!
-        <img
+        {this.props.shouldDisplayImage ? (
+          <img
           className="instructions__logo"
           src="https://uiuc.hack4impact.org/img/colored-logo.png"
           alt="h4i logo"
-        />
+          />
+        ) : (
+          <div />
+        )}
+        <ul>
+          {this.props.items.filter(function(item){
+            return item.length >= 3
+          }).map((item, index) => item.length < 3 ? (
+            <div key={index}/>
+          ) : index % 2 === 0 ? (
+            <li key={index}>{item}</li>
+          ) : (
+            <li key={index}>{item.charAt(0).toUpperCase() + item.slice(1)}</li>
```

Comment:
you can make this more DRY by moving the ternary operator insde the `<li>`, since the containing `<li>` element is repeated:

```jsx
<li>{index % 2 ? item.charAt(0).toUpperCase() + item.slice(1) : item}</li>
```

as shown here, you can also avoid explicitly checking equality to `0` since you can check the truthiness of a number (`0` is false as a conditional)

Reply:
also, you shouldn't use `index` as `key`s : https://egghead.io/lessons/egghead-use-the-key-prop-when-rendering-a-list-with-react